test/extended/operators/olm.go

```diff
@@ -125,16 +125,19 @@ func createResourceByYAML(oc *exutil.CLI, yaml string) {
 	e2e.Logf("Start to create YAML: %s", yaml)
 	_, err := oc.AsAdmin().WithoutNamespace().Run("create").Args("-f", yaml).Output()
 	o.Expect(err).NotTo(o.HaveOccurred())
-	if err != nil {
-		e2e.Failf("Unable to create file:%s", yaml)
-	}
 }
 
 func checkResource(oc *exutil.CLI, commands []string) {
-	output, err := oc.AsAdmin().WithoutNamespace().Run(commands[0]).Args(commands[1 : len(commands)-1]...).Output()
-	if err != nil {
-		e2e.Failf("Unable to get the output:%v", commands)
-	}
+	err := wait.Poll(3*time.Second, 60*time.Second, func() (bool, error) {
+		output, err := oc.AsAdmin().WithoutNamespace().Run(commands[0]).Args(commands[1 : len(commands)-1]...).Output()
```

Comment:
nit: `commands[1:]` is the same as `commands[1:len(commands)-1]`

Reply:
Ah, yes, thank you!